### PR TITLE
fix #730 Notification when no internet access

### LINKED ISSUE #730: Add an alert window when no internet access detected.
The app notifies the user for no internet connection when he has entered all the details for sign up/search. There should be a alert box to notify the user before searching or accessing the feature regarding no internet connection.
I am willing to work on this and would implement a similar alert box a shown in the below screenshot.
screenshot_2018-02-14-01-25-35-411_openfoodfacts github scrachx openfood
screenshot_2018-02-14-01-25-48-003_openfoodfacts github scrachx openfood
screenshot_2018-02-14-01-33-10-777_startup ab nightfoodler

### PR DESCRIPTION
**Pull request by abhinavbajpai2012**
## Fix #730 Add Alert Window when No internet access detected

Added a toast to the activities where ever network access was required.

**Related Issue**
#730 
 

 
 
 
 - [X] Apply the `AndroidStyle.xml` style template to your code in Android Studio.
 
 - [X] If you have multiple commits please combine them into one commit by squashing them.
 
 - [X] Read and understood the contribution guidelines .

### CODE REVIEW

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/SplashActivity.java [hunk 1]
```diff
@@ -40,6 +41,13 @@ public void onCreate(Bundle icicle) {
         settings = getSharedPreferences("prefs", 0);
         mAdditiveDao = Utils.getAppDaoSession(this).getAdditiveDao();
 
+        ConnectivityManager connectivityManager=(ConnectivityManager)getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo netinfo=connectivityManager.getActiveNetworkInfo();
+        if (!(netinfo!=null && netinfo.isConnected()))
+        {
+            Toast.makeText(getApplicationContext(),"No Internet Access",Toast.LENGTH_SHORT).show();
+        }
+
```
Comment (policorn) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/SplashActivity.java]: This code is repeated across the files of this commit. Can you please move out the redundant code to a helper function in other package (i.e. utils)?

**Comment by policorn** [comment]
Hi @abhinavbajpai2012, are you working on it?

**Comment by sachincool** [comment]
It's been 3 days @policorn can i remove the redundancy, refactor it and send the code?  this shouldn't take long

**Comment by policorn** [comment]
Hi @sachincool, yes please, take over and let's move on. Thanks!

**Pull request closed without merging**